### PR TITLE
feat: Add textual results to search box

### PR DESCRIPTION
**Pull request by heyvito**
This adds extra text results to the search box, marking all found terms ✨

### CODE REVIEW

File: static/scss/style.scss [hunk 1]
```diff
@@ -716,6 +717,25 @@ body.showing-search {
...
+                padding-top: 8px;
+                padding-bottom: 10px;
+                font-size: 14px;
+
+                .search-result-match {
+                    color: black;
+                    background-color: yellow;
+                    border-radius: 3px;
+                    padding: 1px 2px;
+                    border: 1px solid #ecb430;
```
Comment (coelhucas) [inline static/scss/style.scss:736]: Won't this need a light variant?
```suggestion
                    color: black;
                    background-color: yellow;
                    border-radius: 3px;
                    padding: 1px 2px;
                    border: 1px solid #ecb430;
```

**Review by coelhucas — COMMENTED** [review]
LGTM, I just left some nitpicks

File: static/scss/style.scss (hunk 1 above)
Comment (coelhucas) [reply static/scss/style.scss:736]: Nevermind, wouldn't make sense here 😅

File: static/js/fts_exec.js [hunk 2]
```diff
@@ -46,8 +46,61 @@ const debounce = (n,t,u) => {var e;return function(){var i=this,o=arguments,a=u&
...
+                .replace(/\s+/g, ' ')
+                .trim();
+
+            const allIndexes = terms.map(t => {
+                const idx = containerText.indexOf(t);
+                if (idx === -1) {
+                    return false;
+                }
+                return { start: Math.max(0, idx - 100), end: idx + 100 };
+            }).filter(Boolean);
```
[the remark this answers is not in the document]
Comment (heyvito) [reply static/js/fts_exec.js:64]: A bit. Could you kindly change your suggestion to split in different lines? 🌻

**Review by coelhucas — APPROVED** [review]

**Review by matheusmedeiros — APPROVED** [review]

**Pull request merged**